.github/workflows/development.yml

```diff
@@ -40,6 +43,8 @@ jobs:
         bundler-cache: true
     
     - name: Run tests
-      timeout-minutes: 5
+      timeout-minutes: 6
+      env:  
+        TRUFFLERUBYOPT: '--experimental-options --engine.MultiTier --engine.Mode=latency'
```

Comment:
Both options shouldn't be used. I'd recommend to remove `--engine.MultiTier`, because there are known issues with it on Native currently.
@gogainda Could you make a PR to do that?
See also https://github.com/oracle/truffleruby/issues/2141#issuecomment-723451953

Reply:
yep, created a PR https://github.com/guard/listen/pull/511